### ISSUE TITLE
Mercedes Class Update/Rework

### ISSUE THREAD

**Issue by CodeSammich** [Maple-Alliance/Issue-Tracker#15]
Submitter: Jovy Cebedo
I play in Reboot GMS. I created my Mercedes for the Link Skill only to be disappointed at how weak and clunky the class actually is. I remember playing Mercedes years back for the Link Skill in normal GMS servers, and it didn't seem too bad compared to all other classes.

The problem now is that the Mercedes class is very underpowered compared to all the classes today. As of right now (April 2021), most classes can up train, mob/farm, and boss with relative ease. When I leveled my Mercedes up, it felt incredibly slow to train. And the class feels very clunky (in terms of skill combos, single-target damage dealing, animations, etc.). The concept of these combo-able abilites with dual bowguns is cool, but I believe it really needs an update. Classes like Kanna, Hayato, Demon Slayer, and Aran have gotten enough updates to keep them interesting to players and to keep them balanced with all other classes. The newest archer class Pathfinder, and the new one soon to be released Kain, are most likely going to be far superior compared to the Mercedes.

Maybe Mercedes isn't too bad near the mid to endgame due to 5th job. Perhaps many Mercedes mains out there don't find the class too bad since they have played the class for a while now. But I found Mercedes to be very boring and clunky during the early game compared to every other class playable.

Simply put; the Mercedes class is outdated, weak, and clunky. They deserve an update/rework to their early game (1st job to pre-5th job). And to an extent, perhaps an update/rework to other classes that don't compare to classes today. Perhaps Hero, Paladin, Wind Archer, Jett, Wild Hunter, and others.

**Comment by CodeSammich** [comment]
Please provide actionable items that are more specific. 

For example: "Please change ___ skill to support ___ functionality".
